solidity/dashboard/src/components/ViewYourLiquidityBalance.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+
+const ViewYourLiquidityBalance = () => {
```

Comment:
Maybe let's create a dir `components/messages` where we can put all `Messages` component and then export them all from the `index.js` file? It will keep components by domain. What do you think?